Week1/Swarnima_Shishodia/Que11.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
+
```

Comment:
Add comment: What this function is doing. Also, it would be good practice to explain your approach in one line or two